src/field/captcha/third_party_captcha.jsx

```diff
@@ -169,23 +182,40 @@ export class ThirdPartyCaptcha extends React.Component {
         if (this.state.retryCount < MAX_RETRY) {
           removeScript(scriptUrl);
           loadScript(scriptUrl, attributes);
-          this.setState((prevState) => ({
+          this.setState(prevState => ({
             retryCount: prevState.retryCount + 1
           }));
           return;
         }
         removeScript(scriptUrl);
         this.changeHandler('BYPASS_CAPTCHA');
       };
-      window[callbackName] = (arkose) => {
+      window[callbackName] = arkose => {
         callback(arkose);
       };
+    } else if (provider === AUTH0_V2_CAPTCHA_PROVIDER) {
+      attributes['error-callback'] = () => {
```

Comment:
Same thing as auth0.js PR (https://github.com/auth0/auth0.js/pull/1382/files#r1433514962). It should be attached to the `onerror` attribute of the script tag.